src/graphql/query/userInfo.graphql

```diff
@@ -9,10 +9,4 @@ query userInfo {
 			balance
 		}
 	}
-	general {
-		loanFindingPage: uiExperimentSetting(key: "loan_finding_page") {
```

Comment:
If you haven't, please add yourself a reminder to remove these old keys from Kiva Admin after the next prod release.

Reply:
Since no more required right now, they are removed from Kiva Admin

Reply:
Can the other experiment settings be removed as well?

Reply:
Yeah, they were inactive... looks like we forgot to removed them while disabling experiments definitively.

Reply:
Please remove if it's safe to do so, and ensure prod behavior is as expected after removing (no new console errors, for example).

Reply:
Yeah, there are not errors about this.

Reply:
What about the other experiment keys removed in this PR?

Reply:
No errors with all the keys removed